bpf/lib/nat.h

```diff
@@ -257,11 +258,11 @@ static __always_inline int snat_v4_new_mapping(struct __ctx_buff *ctx,
 	return !ret ? 0 : DROP_NAT_NO_MAPPING;
 }
 
-static __always_inline int snat_v4_track_local(struct __ctx_buff *ctx,
-					       const struct ipv4_ct_tuple *tuple,
-					       const struct ipv4_nat_entry *state,
-					       enum nat_dir dir, __u32 off,
-					       const struct ipv4_nat_target *target)
+static __always_inline int snat_v4_track_connection(struct __ctx_buff *ctx,
```

Comment:
Should we also rename `snat_v6_track_local`, if only to make sure the two code paths don't diverge too much despite egress gateway still being IPv4-only.

Reply:
Yep, good shout :+1: